utils.sh

```diff
@@ -28,6 +28,18 @@ function create_repo() {
     --repository-name ${repo} > /dev/null
 }
 
+function set_policy() {
+  local region=$1
+  RepositoryPermissionPolicy=$(printenv | grep RepositoryPermissionPolicy | sed 's/.*=//')
```

Comment:
Why is there this complex pipeline here rather than just relying on the `-n` condition on the next line?